core/src/main/java/org/jobrunr/utils/resources/ClassPathResourceProvider.java

```diff
@@ -117,9 +140,10 @@ public void close() throws IllegalStateException {
 
     private FileSystemProvider getFileSystemProviderByScheme(String scheme) {
         switch (scheme) {
-            case "jar": return new JarFileSystemProvider();
-            case "resource": return new ResourcesFileSystemProvider();
-            case "file": return new PathFileSystemProvider();
+            case "jar": return new org.jobrunr.utils.resources.JarFileSystemProvider();
+            case "resource": return new org.jobrunr.utils.resources.ResourcesFileSystemProvider();
+            case "file": return new org.jobrunr.utils.resources.PathFileSystemProvider();
+            case "vfs": return new org.jobrunr.utils.resources.VfsFilesystemProvider();
```

Comment:
Can you import these Provider so they are not fully qualified?

Reply:
fixed